nservicebus/upgrades/nsb6-sc-plugins-1to2.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Nsb6 ServiceControl Plugins Upgrade Version 1 to 2
```

Comment:
Yeah. Not sure how to do that. The plugins are all identified as Nsb6 https://www.nuget.org/packages?q=nsb6